### PR TITLE
Add an analyzer status check into the sketch API client.

### PR DESCRIPTION
**Pull request by kiddinn**
Adds a function `get_analyzer_stats` that goes through all the active timelines in a sketch and checks the status of each of the analyzers that have been run on that timeline.

It returns it as a list of dicts, with information about the analyzer run.

### CODE REVIEW

File: api_client/python/timesketch_api_client/sketch.py [hunk 1]
```diff
@@ -235,6 +235,31 @@ def list_aggregations(self):
             aggregations.append(aggregation_obj)
         return aggregations
 
+    def get_analyzer_status(self):
+        """Returns a list of started analyzers and their status."""
```
Comment (berggren) [inline api_client/python/timesketch_api_client/sketch.py]: Add returns

**Comment by berggren** [comment]
Also: 

```
api_client/python/timesketch_api_client/sketch.py:242:0: C0301: Line too long (82/80) (line-too-long)
api_client/python/timesketch_api_client/sketch.py:241:12: W0621: Redefining name 'timeline' from outer scope (line 28) 
```

File: api_client/python/timesketch_api_client/sketch.py (hunk 1 above)
Comment (kiddinn) [reply api_client/python/timesketch_api_client/sketch.py]: done

**Comment by kiddinn** [comment]
@berggren PTAL

**Pull request merged**